### PR TITLE
update installer script

### LINKED ISSUE #2: Update installer script + README instructions to get the latest release

### PR DESCRIPTION
**Pull request by prnvbn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,4 @@ Coming soon (Make an issue if you want this really quick)
 
 To only download the materials for a particular course, say COMP40009 Computing Practical, run `scientia-cli download 40009`.
 
-## How to install Go
-1. Get the latest tar for your OS from [here](https://go.dev/dl/) by running `wget <link you just copied>`
-2. Run `rm -rf /usr/local/go && tar -C /usr/local -xzf <go1.XX.X.os-arch.tar.gz>`
-3. Add go to your PATH by running `echo "export PATH=\$PATH:/usr/local/go/bin" >> ~/.bashrc`
-4. Run `source ~/.bashrc` to update your PATH
```
Comment (shashwatkansal) [inline README.md:33]: They may be using .zshrc or .bashprofile. Better to leave it generic or link to a site which explains how to add to PATH.

File: linux-installer.sh [hunk 2]
```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
-go build -o scientia-cli
+curl -s https://api.github.com/repos/goDoCer/scientiaCLI/releases/latest \
+    | grep "browser_download_url" \
+    | cut -d : -f 2,3 \
+    | tr -d \" \
+    | wget -O scientia-cli -qi -
```
Comment (shashwatkansal) [inline linux-installer.sh:7]: Can we add execution permissions automatically: `chmod +x ./scientia-cli` ?
```suggestion
    | wget -O scientia-cli -qi -
    | chmod +x ./scientia-cli
```

File: linux-installer.sh [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+    
 sudo mkdir -p /usr/local/bin/scientia-cli
 sudo mv scientia-cli /usr/local/bin/scientia-cli/
```
Comment (shashwatkansal) [inline linux-installer.sh:11]: Might be worth doing a quick checksum validation - there might be automated tools for this?

**Review by shashwatkansal — COMMENTED** [review]
Left a few comments about file executability, PATH. Otherwise looking good!

File: README.md (hunk 1 above)
Comment (prnvbn) [reply README.md:33]: The `linux-installer.sh` print out how to add it to your PATH. Looks like this

```
======================================================================================================
Add scientia-cli to your path. You can do so by running the command depending upon which shell you use

# BASH
echo "export PATH=\$PATH:/usr/local/bin/scientia-cli" >> ~/.bashrc

# ZSH 
echo "export PATH=\$PATH:/usr/local/bin/scientia-cli" >> ~/.zshrc

# FISH 
set -U fish_user_paths /usr/local/go/bin $fish_user_paths

Run source ~/.bashrc afterwards
======================================================================================================
```

File: linux-installer.sh (hunk 2 above)
Comment (prnvbn) [reply linux-installer.sh:7]: not required, what we upload to the releases page is already an executable

File: linux-installer.sh (hunk 3 above)
Comment (Akshat-Tripathi) [reply linux-installer.sh:11]: Yeah sure, we'd need to add the checksum in the release notes, and maybe run it after downloading the file too?

Comment (prnvbn) [reply linux-installer.sh:11]: seems a bit overkill 😅, no?

**Review by shashwatkansal — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (shashwatkansal) [reply README.md:33]: At the end, you say `source ~/.bashrc`. Maybe don't include that or make it generic?

**Pull request merged**